LICENSE

```diff
@@ -1 +0,0 @@
-COPYING
```

Comment:
Could we keep this file, and just have it include the licensing information from the README, and link to it from the README instead?

Reply:
OK, done.